network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/app_install_hero.html

```diff
@@ -0,0 +1,63 @@
+>
+        <div class="tw-absolute tw-inset-0">
+            {% include "./responsive_screen_wide_image.html" with image_object=page.hero_background %}
+            <div class="tw-absolute tw-inset-0" aria-hidden="true"></div>
+        </div>
+    <div class="tw-relative container">
+        <div class="row">
+            <div class="tw-px-6 small:tw-px-24 large:tw-container large:tw-px-0">
+                <div class="tw-grid tw-gap-x-24 tw-grid-cols-1 tw-px-8 medium:tw-grid-cols-2 ">
+                    <div class="tw-flex tw-flex-col tw-justify-center">
+                        <h2 class="tw-mb-8 tw-text-white tw-h1-heading tw-font-light tw-text-4xl large:tw-text-5xl">
+                            {{ page.hero_heading }}
+                        </h2>
+                        <p class="tw-body-large tw-text-white tw-font-light tw-pb-4">
+                            {{ page.hero_subheading }}
+                        </p>
+                        {% with download_btn_classes="tw-max-w-max tw-btn tw-bg-white hover:tw-bg-gray-20 tw-transition-colors tw-border-black tw-border-2" btn_image_classes="tw-mr-2 tw-inline" %}
+                            <div class="download-btn-wrapper">
+                                <div class="btn-container tw-mt-2 tw-flex tw-flex-wrap tw-gap-8">
+                                    {% if page.button1_text and page.button1_download_link %}
+                                        <a class="{{ download_btn_classes }}" href="{{ page.button1_download_link }}" target="_blank">
+                                            {% if page.button1_image %}
+                                                {% image page.button1_image original as btn1_image %}
+                                                <img src="{{ btn1_image.url }}" alt="{{ page.button1_image_alt }}" class="{{ btn_image_classes }}" width="24" height="24">
+                                            {% endif %}
+                                            {{ page.button1_text }}
+                                        </a>
+                                    {% endif %}
+                                    {% if page.button2_text and page.button2_download_link %}
+                                        <a class="{{ download_btn_classes }}" href="{{ page.button2_download_link }}" target="_blank">
+                                            {% if page.button2_image %}
+                                                {% image page.button2_image original as btn2_image %}
+                                                <img src="{{ btn2_image.url }}" alt="{{ page.button2_image_alt }}" class="{{ btn_image_classes }}" width="24" height="24">
+                                            {% endif %}
+                                            {{ page.button2_text }}
+                                        </a>
+                                    {% endif %}
```

Comment:
If we re-work the button data, this is where we can condense the code down to a loop and repeat less

Reply:
This has been updated in the latest commit to use the new `AppInstallDownloadButtonBlock` and to loop through them. Thanks for the suggestion Rob!